web/src/components/Plugin/PluginPage.tsx

```diff
@@ -66,6 +66,8 @@ const PluginPage: React.FC<Props> = ({
   const { formatMessage } = useIntl();
   const [form] = Form.useForm();
   const [pluginList, setPluginList] = useState<PluginComponent.Meta[]>([]);
+  const [enablePluginsList, setEnablePluginsList] = useState<PluginComponent.Meta[]>([]);
```

Comment:
enablePluginsList can be obtained from the `pluginList` after calculation, no need to define a new state.